topo_processor/cog/create_cog.py

```diff
@@ -0,0 +1,33 @@
+import asyncio
+import os
+
+from linz_logger import get_log
+
+from topo_processor.util.time import time_in_ms
+
+
+def docker_gdal():
+    return f"docker run \
+        --user {os.geteuid()}:{os.getegid()} \
+        --volume {os.getcwd()}:{os.getcwd()} \
```

Comment:
this path needs to include both the input and output directories

Reply:
This isnt going to work, you need to find the best paths for input and output, CWD is not needed.

```python

volumes = []
if input.startsWith(output):
  volumes.push(output)
else if output.startsWith(input):
  volumes.push(input)
else if output == input:
 volumes.push(input)
else:
  volumes.push(input, output)
```